data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/TopicConfig.java

```diff
@@ -73,22 +74,18 @@ public class TopicConfig {
     private Duration threadWaitingTime = DEFAULT_THREAD_WAITING_TIME;
 
     @JsonProperty("max_partition_fetch_bytes")
-    private Integer maxPartitionFetchBytes = DEFAULT_MAX_PARTITION_FETCH_BYTES;
+    private String maxPartitionFetchBytes = DEFAULT_MAX_PARTITION_FETCH_BYTES;
```

Comment:
Data Prepper should be able to deserialize these natively. But, we can always clean this up later.